src/PowerShellEditorServices/Symbols/PesterDocumentSymbolProvider.cs

```diff
@@ -131,6 +131,23 @@ private static PesterSymbolReference ConvertPesterAstToSymbolReference(ScriptFil
                 pesterCommandAst.Extent
             );
         }
+
+        private static bool TryGetTestNameArgument(CommandElementAst commandElementAst, out string testName)
+        {
+            testName = null;
+
+            if (commandElementAst is StringConstantExpressionAst testNameStrAst)
+            {
+                testName = testNameStrAst.Value;
+                return true;
+            }
+            else if (commandElementAst is ExpandableStringExpressionAst)
```

Comment:
No need for the `else` here -- above clause always returns

Reply:
This could be a [pattern match](https://docs.microsoft.com/en-us/dotnet/csharp/pattern-matching) too since it just tests type overloads. But I'm easy either way :)